video/out/vo_dmabuf_wayland.c

```diff
@@ -354,6 +344,35 @@ static int preinit(struct vo *vo)
         return VO_ERROR;
     }
 
+
+    if (vo->wl->single_pixel_manager) {
+#if HAVE_WAYLAND_PROTOCOLS_1_27
+        p->solid_buffer = wp_single_pixel_buffer_manager_v1_create_u32_rgba_buffer(
+            vo->wl->single_pixel_manager, 0, 0, 0, UINT32_MAX); /* R, G, B, A */
+#endif
+    } else {
+#if HAVE_WAYLAND_PROTOCOLS_1_27
+        MP_INFO(vo->wl,
+            "Compositor doesn't support the %s protocol, falling back to using wl-shm",
+            wp_single_pixel_buffer_manager_v1_interface.name);
+#endif
```

Comment:
Not exactly what I wanted. This check should be in `wayland_common.c` under where the `if (wl->content_type_manager) {` block is.